[2023-08-13T09:42:35.080+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tune_dag.tune_task manual__2023-08-13T09:34:18.959133+00:00 [queued]>
[2023-08-13T09:42:35.084+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tune_dag.tune_task manual__2023-08-13T09:34:18.959133+00:00 [queued]>
[2023-08-13T09:42:35.084+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-08-13T09:42:35.090+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): tune_task> on 2023-08-13 09:34:18.959133+00:00
[2023-08-13T09:42:35.094+0000] {standard_task_runner.py:57} INFO - Started process 311 to run task
[2023-08-13T09:42:35.096+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'tune_dag', 'tune_task', 'manual__2023-08-13T09:34:18.959133+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/scripts/tune_ml.py', '--cfg-path', '/tmp/tmpd_5nstam']
[2023-08-13T09:42:35.097+0000] {standard_task_runner.py:85} INFO - Job 136: Subtask tune_task
[2023-08-13T09:42:35.113+0000] {task_command.py:410} INFO - Running <TaskInstance: tune_dag.tune_task manual__2023-08-13T09:34:18.959133+00:00 [running]> on host 30ba972171bc
[2023-08-13T09:42:35.142+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thanhdong18891@gmail.com' AIRFLOW_CTX_DAG_OWNER='DongNT17' AIRFLOW_CTX_DAG_ID='tune_dag' AIRFLOW_CTX_TASK_ID='tune_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-13T09:34:18.959133+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-13T09:34:18.959133+00:00'
[2023-08-13T09:42:35.143+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-08-13T09:42:35.143+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', ' bash -i /opt/***/dags/scripts/train.sh ']
[2023-08-13T09:42:35.151+0000] {subprocess.py:86} INFO - Output:
[2023-08-13T09:42:35.153+0000] {subprocess.py:93} INFO - bash: cannot set terminal process group (-1): Inappropriate ioctl for device
[2023-08-13T09:42:35.153+0000] {subprocess.py:93} INFO - bash: no job control in this shell
[2023-08-13T09:42:36.187+0000] {subprocess.py:93} INFO - # conda environments:
[2023-08-13T09:42:36.187+0000] {subprocess.py:93} INFO - #
[2023-08-13T09:42:36.187+0000] {subprocess.py:93} INFO - base                     /home/***/conda
[2023-08-13T09:42:36.187+0000] {subprocess.py:93} INFO - training_env          *  /home/***/conda/envs/training_env
[2023-08-13T09:42:36.187+0000] {subprocess.py:93} INFO - 
